tests/features/capsulahub_run.feature

```diff
@@ -4,54 +4,68 @@ Feature: Build and Run commands for CapsulaHub CLI
 
 #______________________________________POSITIVE______________________________________
 
   #1
-  Scenario: Run CapsulaHub instance with specifying valid arguments (check for every <configProvider>)
-    Given a valid token with configuration
-    When  I run the command `capsulahub run --token="token" --configProvider="configProvider" --dispatcherUrl="dispatcherUrl" --port="port"` #add dispatcherUrl only for scalecube provider
-    And   "configProvider" is one from the available providers
-    Then  an workspace is created
-    And   workspace is running on localhost on the specified port
-    And   the specified configProvider is used
+  Scenario: Run CapsulaHub instance with specifying valid arguments (check for HttpFile provider)
+    Given a configuration that includes component A
+    And   token B that allow access to this configuration using HttpFileConfigurationProvider
+    When  I run the command `capsulahub run --token="tokenB" --configProvider="HttpFile"  --port="6666"`
+    And   the app is running on the specified port "6666"
+    And   I open the app in the browser
+    Then  I see component A rendered in the app
+    And   HttpFile is the provider used to get configuration of token B
 
   #2
-  Scenario: Run CapsulaHub instance without specifying the port and configProvider
-    Given a valid token with configuration
-    When  I run the command `capsulahub run --token="token"`
-    Then  an workspace is created
-    And   workspace is running on localhost on the default port
-    And   httpFile is the configuration provider used
+  Scenario: Run CapsulaHub instance with specifying valid arguments (check for LocalFile, Scalecube, HttpServer and LocalStorage provider)
+    Given a configuration that includes component A
+    And   token B that allow access to this configuration using <configProvider> and <dispatcherUrl>
+          |<configProvider>                   |<dispatcherUrl>
+          |LocalFileConfigurationProvider     |(empty)
+          |ScalecubeConfigurationProvider     |'url123'
+          |HttpServerConfigurationProvider    |(empty)
+          |LocalStorageConfigurationProvider  |(empty)
+    When  I run the command `capsulahub run --token="tokenB" --configProvider="configProvider"  --port="6666"`
+    Then  an workspace is created using right configuration provider
 
   #3
+  Scenario: Run CapsulaHub instance without specifying the port and configProvider
+    Given a configuration that includes component A
+    And   token B that allow access to this configuration
+    When  I run the command `capsulahub run --token="tokenB"`
+    And   the app is running on the default port "5555"
+    And   I open the app in the browser
+    Then  I see component A rendered in the app
+    And   HttpFile is the provider used to get configuration of token B
+
+  #4
   Scenario: Run CapsulaHub instance with the same token for different configProvider on two different ports
```

Comment:
suggest to remove the part about "different configProvider", because we will not be able to open them on different providers given the fact that we open in browser only in httpFile provider.